integration-libs/cdc/src/auth/facade/cdc-js.service.ts

```diff
@@ -38,7 +69,8 @@ export class CdcJsService implements OnDestroy {
     protected auth: AuthService,
     protected zone: NgZone,
     protected userService: UserService,
-    @Inject(PLATFORM_ID) protected platform: any
+    @Inject(PLATFORM_ID) protected platform: any,
```

Comment:
If you have @Inject() in one of the constructor parameters it won't work in the production build if you use overloads. In this case, you have to provide just one constructor and handle deprecations in comments.